roles/docker/templates/docker.service.j2

```diff
@@ -30,6 +30,9 @@ ExecStart={{ docker_bin_dir }}/docker{% if installed_docker_version.stdout|versi
           $DOCKER_NETWORK_OPTIONS \
           $DOCKER_DNS_OPTIONS \
           $INSECURE_REGISTRY
+{% if docker_orphan_clean_up == "true" %}
```

Comment:
Would this make more sense as a separate drop-in that's copied over conditionally? Similar to docker storage options and proxy